.github/workflows/validate-metadata.yml

```diff
@@ -2,16 +2,34 @@ name: Fastlane Metadata
 on:
   push:
     branches:
-      - main
+      - 'upload-to-playstore'
 jobs:
-  validate:
-    # required to run on Linux because this is a docker container action
-    name: validate
+  marketing:
     runs-on: ubuntu-latest
     steps:
+     - name: Decrypt API JSON file
+       run: cd ./packages/smooth_app/android/fastlane/envfiles && chmod +x ./decrypt_secrets.sh && ./decrypt_secrets.sh
+       env:
+        API_JSON_FILE_DECRYPTKEY: ${{ secrets.API_JSON_FILE_DECRYPTKEY }}
+        STORE_JKS_DECRYPTKEY: ${{ secrets.STORE_JKS_DECRYPTKEY }}
```

Comment:
Lets put the validation before the encryption

Reply:
done